modules/programs/zellij.nix

```diff
@@ -49,9 +49,28 @@ in {
       default = false;
     };
 
-    enableFishIntegration = mkEnableOption "Fish integration" // {
+    enableFishIntegration = mkEnableOption
+      "Fish integration (enables both enableFishAutoStart and enableFishCompletions)"
+      // {
+        default = false;
+      };
+    enableFishCompletions = mkEnableOption "load zellij completions" // {
       default = false;
     };
+    enableFishAutoStart =
+      mkEnableOption "autostart zellij in interactive sessions" // {
+        default = false;
+      };
+    autoStartAttachIfSessionExists = mkEnableOption
+      "If the zellij session already exists, attach to the default session. (requires enableFishAutoStart)"
+      // {
+        default = false;
+      };
+    autoStartExitShellOnZellijExit = mkEnableOption
+      "When zellij exits, exit the shell as well. (requires enableFishAutoStart)"
```

Comment:
mkEnableOption uses a default description text template of the form `Whether to <your string you pass>`. It would be great if you can adapt your text so that the whole sentence makes sense. 

Or you can simply rely on `mkOption` with a bool type instead. And fully customize the description text.

Reply:
Also good call - updated all descriptions :)